docs/resources/Channel.md

```diff
@@ -552,14 +552,51 @@ A thread member is used to indicate whether a user has joined a thread or not.
 
 Embed types are "loosely defined" and, for the most part, are not used by our clients for rendering. Embed attributes power what is rendered. Embed types should be considered deprecated and might be removed in a future API version.
```

Comment:
Considering that the rendering of automod embeds highly relies on the embed type, maybe this sentence could now be considered inaccurate and be removed?